Packs/MicrosoftGraphMail/Integrations/MicrosoftGraphListener/MicrosoftGraphListener.yml

```diff
@@ -111,6 +111,7 @@ configuration:
   required: true
   type: 0
   section: Collect
+  additionalinfo: Make sure you are logged in to this email during configuration and authentication, otherwise it will not work.
```

Comment:
Maybe something like:
 "As this is a single-user integration, please make sure to authenticate with the same user configured" ?

Reply:
@ShirleyDenkberg WDYT?

Reply:
"As this is a single-user integration, make sure to authenticate with the same configured user."

Reply:
@yucohen @ShirleyDenkberg Is the proposed sentence clear that the user should be connected to the email he enters in the configuration?